app/helper/rtForm.php

```diff
@@ -1075,6 +1080,130 @@ public function get_select( $attributes = '' ) {
 		public function display_select( $args = '' ) {
 			echo $this->get_select( $args );
 		}
+
+		/**
+		 * Generate rtmedia html button in admin options.
+		 *
+		 * @access protected
+		 *
+		 * @param $attributes
+		 *
+		 * @return string
+		 * @throws rtFormInvalidArgumentsException
+		 */
+		protected function generate_button( $attributes ) {
+
+			$element = 'rtButton';
+			if ( is_array( $attributes ) ) {
+
+				/* Starting the input tag */
+				$html = '<input type="button" ';
+
+				/* generating attributes */
+				$html .= $this->processAttributes( $element, $attributes );
+
+				/* ending the tag */
+				$html .= ' />';
+
+				if ( isset( $attributes['label'] ) ) {
+					if ( isset( $attributes['labelClass'] ) ) {
+						$html = $this->enclose_label( $element, $html, $attributes['label'], $attributes['labelClass'] );
+					} else {
+						$html = $this->enclose_label( $element, $html, $attributes['label'] );
+					}
+				}
+
+				if ( isset( $attributes['show_desc'] ) && $attributes['show_desc'] ) {
+					$html .= $this->generate_element_desc( $attributes );
+				}
+
+				return $html;
+			} else {
+				throw new rtFormInvalidArgumentsException( 'attributes' );
+			}
+		}
+
+		/**
+		 * Get rtmedia html button in admin options.
+		 *
+		 * @access public
+		 *
+		 * @param string/array $attributes
+		 *
+		 * @return string
+		 * @throws rtFormInvalidArgumentsException
+		 */
+		public function get_button( $attributes = '' ) {
+
+			return $this->generate_button( $attributes );
+		}
+
+		public function display_button( $args = '' ) {
+
+			echo $this->get_button( $args );
+		}
+
+		/**
+		 * Generate rtmedia html file input in admin options.
+		 *
+		 * @access protected
+		 *
+		 * @param $attributes
+		 *
+		 * @return string
+		 * @throws rtFormInvalidArgumentsException
+		 */
+		protected function generate_file_input( $attributes ) {
+
+			$element = 'rtFileInput';
+			if ( is_array( $attributes ) ) {
```

Comment:
Avoid large `if` block. Throw an exception if `$attributes` is not an array. else further code will execute. So you don't need `else` condition.